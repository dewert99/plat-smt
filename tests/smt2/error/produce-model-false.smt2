(set-option :produce-models false)
(declare-const a Bool)
(assert a)
(check-sat)
(get-model)
(get-value (a))