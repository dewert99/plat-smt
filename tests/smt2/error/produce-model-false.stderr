The option `:produce-models` must be set to true at 5:2
(get-model)
 ^

The option `:produce-models` must be set to true at 6:2
(get-value (a))
 ^

